### PR TITLE
Support new hf.co/ format

### PR DESCRIPTION
**Pull request by ericcurtin**
This new format for hf has been implemented by Ollama.

### CODE REVIEW

**Comment by ericcurtin** [comment]
This doesn't work that well, but this is up for grabs if someone can figure it out

File: ramalama/cli.py [hunk 1]
```diff
@@ -688,13 +688,38 @@ def dry_run(args):
...
+
+    # Check if quantization is specified, else default to Q4_K_M
+    if ':' in model_info:
+        model_name, quantization = model_info.split(':')
+    else:
+        model_name = model_info
+        quantization = 'Q4_K_M'  # Default quantization scheme
+
+    # Construct the new format
+    return f"hf://{username}/{model_name}/{model_name}-{quantization}.gguf"
```
Comment (swarajpande5) [inline ramalama/cli.py:708]: @ericcurtin I would like to take this up, the new format which is being returned might not always be correct. 
For example, running the following, 
```
ramalama pull hf.co/instructlab/merlinite-7b-lab-GGUF
```
will use 
```
hf://instructlab/merlinite-7b-lab-GGUF/merlinite-7b-lab-GGUF-Q4_K_M.gguf
```
And the above path does not exist.

Comment (ericcurtin) [reply ramalama/cli.py:708]: Yeah... Maybe there's a REST API to get the real filename if you pass it instructlab/merlinite-7b-lab-GGUF and Q4_K_M or something like that but I'm really guessing...

Comment (swarajpande5) [reply ramalama/cli.py:708]: Yes, there should be one to get the complete path. I'll explore this :)

**Comment by rhatdan** [comment]
Any update on this?

**Comment by rhatdan** [comment]
@ericcurtin @swarajpande5 Should this close?

**Comment by ericcurtin** [comment]
Closing...

**Pull request closed without merging**